src/commands/validate-config.js

```
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
+
+const Config = require('../lib/config');
+const logger = require('../lib/logger');
+
+const configSchema = require('../../schemas/config.schema.json');
+
+const main = async (argv) => {
+
+	const config = new Config({
+		source: argv.config,
+		schema: configSchema
+	});
+
+	try {
+		await config.load();
+
+		logger.info(`Config: Read from ${config.sourceDescription}\n`);
+		logger.success('Config is valid');
+	} catch (err) {
+		logger.error('Config is invalid');
+		logger.error(err.message);
```

Throw these errors instead of logging them here.